assets/src/components/propertiesPanel/blockWaiverBanner.tsx

```diff
@@ -0,0 +1,128 @@
+import React from "react"
+import { alertCircleIcon } from "../../helpers/icon"
+import { BlockWaiver } from "../../realtime"
+
+interface Props {
+  blockWaiver: BlockWaiver
+}
+
+export const hours12 = (hours24plus: number): number => {
+  if (hours24plus === 24) {
+    return 12
+  }
+  if (hours24plus > 24) {
+    return hours24plus - 24
+  }
+  if (hours24plus > 12) {
+    return hours24plus - 12
+  }
+  return hours24plus
+}
+
+const ampm = (hours24plus: number): string =>
+  hours24plus >= 12 && hours24plus < 24 ? "pm" : "am"
+
+const zeroPad = (time: number): string => (time < 10 ? `0${time}` : `${time}`)
+
+export const dateFromEpochSeconds = (time: number): Date =>
+  new Date(time * 1_000)
+
+export const formattedTime = (date: Date): string => {
+  const hours24 = date.getHours()
+  return `${hours12(hours24)}:${zeroPad(date.getMinutes())}${ampm(hours24)}`
+}
+
+export const formatEpochSeconds = (epochSeconds: number): string => {
+  const date = dateFromEpochSeconds(epochSeconds)
+  return formattedTime(date)
+}
+
+export const nowEpochSeconds = (now = Date.now()): number =>
+  Math.floor(now / 1_000)
+
+export enum CurrentFuturePastType {
+  Current = 1,
+  Future,
+  Past,
+}
+
+export const currentFuturePastType = ({
```

Comment:
Some things in this file don't need to be exported anymore.

Reply:
Removed